tests/test_multistore_file.py

```diff
@@ -27,6 +34,32 @@ def filename(self):
 
 
 class MultistoreFileTests(unittest.TestCase):
+    def tearDown(self):
+        try:
+            os.unlink(FILENAME)
+        except OSError:
+            pass
+
+    def setUp(self):
```

Comment:
`setUp` and `tearDown` are new (i.e. not ported directly).

Why are they introduced?